python/cuml/cuml/internals/array.py

```diff
@@ -1172,6 +1172,19 @@ def from_input(
         if (
             not fail_on_order and order != arr.order and order != "K"
         ) or make_copy:
+            if make_copy:
+                info(
```

Comment:
There are other places where a copy can happen, specifically lines 1072, potentially 1104 (unless the dataframe is fully contiguous) and potentialy 1160, not sure if we need all of them but it would be good to cover most of the cases.

Reply:
Regarding line 1072, the copy happens when the function is called with `deepcopy=True`, which means that a copy was required anyway. The user cannot do much about it. Do we want to log this as well?

Regarding line 1104, can the data in the dataframe be non-contiguous? Not very familiar into the ways to check for the internals of cuDF, but it might be something we want to check too. However again, the user cannot do much about this to avoid a copy.

Regarding line 1160, unless mistaken this section of code should be in charge of data type conversions. Do we want to log this too? But, then to be consistent we would have to double check that every data type conversions is being caught. Given the time left for release, it might be a bit short.